pkg/controller/controller.go

```diff
@@ -120,9 +120,9 @@ func (c *Controller) makeClusterConfig() cluster.Config {
 }
 
 func (c *Controller) initCRD() error {
-	err := k8sutil.CreateCRD(c.KubeExtCli)
+	err := k8sutil.CreateCRD(c.KubeExtCli, api.CRDName, api.CRDResourceKind, api.CRDResourcePlural)
```

Comment:
Can you also change `CRDName` to `EtcdClusterCRDName`?

Reply:
yeah